### DISCUSSION TITLE
the KR normalization

### DISCUSSION THREAD

**Discussion by Nuturetree** [paulsengroup/hictk#307] [Q&A] [answered]
Does hictk support KR standardization?

**Comment by robomics** [comment, marked as answer]
It does support it, in the sense that if a .hic or .[m]cool file contains KR weights, hictk can read and use those weights (e.g. when dumping interactions).

However, there are no plans to add KR support to `hictk balance`.
The reason being that I do not believe there is a way to implement KR efficiently, so a hictk implementation would be about as slow as e.g. `juicer_tools`.

If you have some tool that expects KR normalized weights, and that does not allow you to customize the weight names, what you could do is run one of the `hictk balance` subcommands and tell hictk to name the weights "KR".

Example:
```bash
hictk balance scale file.hic --name KR 
```

If you decide to use ICE, you most likely have to specify `--no-rescale-weights` as well (and if you are balancing .[m]cool files that have already been balanced by cooler or hictk, you may want to pass `--no-create-weight-link` too).
